### ISSUE TITLE
Not getting all variables when running dump

### ISSUE THREAD

**Issue by joharohl** [jagheterfredrik/esphome-rego1000#26]
I'm well aware that my issue is probably not related to the code itself, but hoping you can help anyway.
I have a Rego1000 with version 3.10.0 and as that versions variables are not present I'm trying to dump them.

However I get different amount of variables back everytime I run the dump program.
I'm guessing I'ts down to hardware eventhough I have checked a lot and everything seems ok. 

Any guesses as to why it's not working correctly and what can I do to troubleshoot?

**Comment by jagheterfredrik** [comment]
Yeah the dumping is kind of unreliable. I sometimes have to run it multiple times, starting from different offsets and stitch it together

**Comment by joharohl** [comment]
Ok. That is somewhat relieving :) How can change the start address?

**Comment by jagheterfredrik** [comment]
You can mess with the [`remote_rd_ptr`](https://github.com/jagheterfredrik/esphome-rego1000/blob/db9c2045bdeb1eab13fa395a433a80c8071f7368/regovardump.h#L30), setting it to a multiple of 0x4e20. Or just keep running it until you get all variables
